vite.config.mjs

```diff
@@ -1,16 +1,29 @@
 import { resolve } from 'path'
 
 import vue from '@vitejs/plugin-vue'
-import { defineConfig } from 'vite'
+import { defineConfig, normalizePath } from 'vite'
 import cssInjectedByJsPlugin from 'vite-plugin-css-injected-by-js'
+import { viteStaticCopy } from 'vite-plugin-static-copy'
 
 // Set to this to the name of this collection of components
 // This must match node-red-dashboard-2.widgets[libraryName] in package.json
 const LIBRARY_NAME = 'ui-example'
 
 // https://vitejs.dev/config/
 export default defineConfig({
-    plugins: [vue(), cssInjectedByJsPlugin()],
+    plugins: [
+        vue(),
+        cssInjectedByJsPlugin(),
```

Comment:
purpose of this plugin?

Reply:
You may tell me; this was already part of the (original) code! 😉

Reply:
With a bit of investigation: From [this plugin's documentation](https://www.npmjs.com/package/vite-plugin-css-injected-by-js):

> A Vite plugin that takes the CSS and adds it to the page through the JS. For those who want a single JS file.

Thus ... very useful and quite relevant for the NR use case!